examples/providers/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "examples-providers"
+
+publish.workspace = true
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+authors.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+
+[dev-dependencies]
+alloy-network.workspace = true
+alloy-node-bindings.workspace = true
+alloy-contract.workspace = true
+alloy-primitives.workspace = true
+alloy-provider = { workspace = true, features = ["pubsub", "ws"] }
+alloy-pubsub.workspace = true
+alloy-sol-types = { workspace = true, features = ["json"] }
+alloy-rpc-types.workspace = true
+alloy-rpc-client.workspace = true
+alloy-rpc-trace-types.workspace = true
+alloy-signer-wallet.workspace = true
+alloy-transport.workspace = true
+alloy-transport-http.workspace = true
+alloy-transport-ws.workspace = true
+alloy-transport-ipc.workspace = true
```

Comment:
Please use the new Alloy namespace (see: https://github.com/alloy-rs/examples/pull/11)

Reply:
@zerosnacks need some help here. I'm using the alloy namespace now. However, the I'm unable to enable the "pubsub" feature on rpc-client to make `connect_pubsub` method available on `RpcClient` in the `ipc`, `ws` `ws_with_auth` examples

Reply:
Looking into it now, it is likely related to missing feature flags in the `alloy` crate

Reply:
Thanks, let me know if you find a fix. Facing the same issue in the subscription examples

Reply:
If something is a blocker please use the original direct import for that specific issue until a fix is in place

Reply:
I was able to track / fix this in https://github.com/alloy-rs/alloy/pull/392 but it will require a larger discussion on what / how `alloy` wants to expose these feature flags.

@yash-atreya  If you run into any more missing flags please add them to this ticket: https://github.com/alloy-rs/alloy/issues/391